src/app/(main)/profile/myActivitiesStatus/page.tsx

```
@@ -187,61 +201,47 @@ export default function ReservationStatusPage() {
 
           statusBadgeData[date] = dateCounts;
-          console.log(`✅ ${date} 최종 상태:`, JSON.stringify(dateCounts, null, 2));
         } catch (err) {
           console.warn(`Failed to load status for ${date}:`, err);
         }
       }
 
-      // 전역 변수에 저장 (기존 방식과 호환성 유지)
+      // 🌐 전역 변수에 저장 (기존 방식과 호환성 유지)
+      // 캘린더 컴포넌트에서 이 데이터를 참조하여 뱃지 표시
       (window as any).statusBadgeData = statusBadgeData;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)